README.md

```diff
@@ -4,26 +4,40 @@
 
 ## Overview
 
-The `renv` package is a new effort to bring private project-local R libraries
-to your projects. The goal is for `renv` to be a robust, stable replacement for
-the [Packrat](https://rstudio.github.io/packrat/) package, with fewer surprises
-and better default behaviors.
+The renv package creates **r**eproducible **env**ironments that increase the
```

Comment:
I want to avoid over-promising on reproducibility. I'd prefer language like "`renv` helps you to produce reproducible research by <...>", rather than giving the impression that `renv` could be a panacea for all issues in reproducibility.

Reply:
I thought this language was fairly mild - I just say "increase the chances"